backend/lib/security/jose.js

```diff
@@ -69,8 +74,16 @@ module.exports = sessionSecret => {
       }
       return jwtSign(payload, secretOrPrivateKey, options)
     },
-    verify (token, options) {
-      return jwtVerify(token, sessionSecret, options)
+    async verify (token, options) {
+      let lastError
+      for (const sessionSecret of sessionSecrets) {
+        try {
+          return await jwtVerify(token, sessionSecret, options)
+        } catch (error) {
+          lastError = error
+        }
+      }
+      throw lastError
```

Comment:
It could be missleading to return the last error. Assume both secrets are in general invalid I think the first error is more significant. I would return a new Error.
```
async verify (token, options) {
  let firstError
  for (const sessionSecret of sessionSecrets) {
    try {
      return await jwtVerify(token, sessionSecret, options)
    } catch (err) {
      firstError ??= err
    }
  }
  throw new Error('Token cannot be verified with any of the session secrets')
}
```